src/types/index.ts

```diff
@@ -10,3 +10,45 @@ export interface ChainProperty {
 }
 
 export type ExtrinsicPayload = SubmittableExtrinsic<"promise">;
+
+export type Encrypted = { iv: string; data: string };
+
+export interface AccountData {
+  isDev: boolean;
+  alias: string;
+  mnemonic: string | Encrypted;
+  address: string;
+}
+
+export interface ContractData {
+  language: "ask" | "ink";
+  name: string;
+  build?: BuildData;
+  deployments: DeploymentData[] | [];
+}
+
+export interface BuildData {
+  timestamp: number;
+  artefactsPath: string;
+}
+
+export interface DeploymentData {
+  timestamp: number;
+  networkUrl: string;
+  deployerAlias: string;
+  address: string;
+}
+export interface SwankyConfig {
+  node: {
+    polkadotPalletVersions: string;
+    localPath: string;
+    supportedInk: string;
+  };
+  accounts: AccountData[];
+  contracts: Record<string, ContractData> | Record<string, never>;
```

Comment:
How can we make this iterable?
For example, we want to iterate over contracts here https://github.com/AstarNetwork/swanky-cli/pull/30/files#diff-65b1b435366fc386594237e408085f244b18ecd5a4a1166a37764c232e385b3eR74

Reply:
You mean the record?
It's meant to be accessed by id (name in this case), but if you need to iterate, then `Object.values(contracts)` will give you `ContractData[]`

Reply:
Oh, I see. that sounds good!